src/components/TodoHeader/TodoHeader.tsx

```diff
@@ -0,0 +1,77 @@
+import React, {
+  FormEvent, useEffect, useRef, useState,
+} from 'react';
+import cn from 'classnames';
+import { ErrorMessages } from '../../types/ErrorMessages';
+
+type Props = {
+  onError: (value: ErrorMessages | null) => void;
+  createTodo: (title: string) => void;
+  isInputDisabled: boolean;
+  activateToggleAll: boolean;
+  setLoadingAll: (value: boolean) => void;
+  toggleAll: () => void;
+  focusTitleInput: boolean;
+};
+
+export const TodoHeader: React.FC<Props> = ({
+  onError,
+  createTodo,
+  isInputDisabled,
+  activateToggleAll,
+  setLoadingAll,
+  toggleAll,
+  focusTitleInput,
+}) => {
+  const [newTodoTitle, setNewTodoTitle] = useState<string>('');
+  const inputTitle = useRef<HTMLInputElement>(null);
+
+  const addTodo = (e: FormEvent) => {
+    e.preventDefault();
+    const todoName = newTodoTitle.trim();
+
+    if (todoName === '') {
+      onError(ErrorMessages.emptyTitle);
+
+      return;
+    }
+
+    createTodo(todoName);
+    setNewTodoTitle('');
+  };
```

Comment:
why not focus input after adding new todo?
```suggestion
    setNewTodoTitle('');
    inputTitle.current?.focus();
  };
```